certbot_dns_ionos/dns_ionos.py

```diff
@@ -114,22 +114,21 @@ def _api_request(self, type, action, data = None):
             headers = self.headers
             headers['Content-Type'] = 'application/json'
             resp = requests.put(url, headers=headers, data=json.dumps(data))
-        elif type == 'patch':
+        elif type == 'post':
             headers = self.headers
             headers['Content-Type'] = 'application/json'
-            resp = requests.patch(url, headers=headers, data=json.dumps(data))
+            resp = requests.post(url, headers=headers, data=json.dumps(data))
         elif type == 'delete':
             resp = requests.delete(url, headers=self.headers)
         else:
             raise errors.PluginError(
                 "HTTP Error during request. Unknown type {0}".format(type)
             )
-        logger.debug("API REquest to URL: %s", url)
-        if resp.status_code != 200:
+        logger.debug("API Request to URL: %s", url)
+        if not resp.ok:
```

Comment:
The newly introduced POST requets is responded to with `201` by the current IONOS API appearantly.